### ISSUE TITLE
`Failed to create worker thread` - weblogic-operator-webhook

### ISSUE THREAD

**Issue by audacioustux** [verrazzano/verrazzano#5758]
Stuck in CrashLoopBackOff on weblogic-operator-webhook pod

```
Launching the Domain Custom Resource Conversion Webhook for Oracle WebLogic Server Kubernetes Operator...                                                                                                                                                         
[0.007s][warning][os,thread] Failed to start thread "GC Thread#0" - pthread_create failed (EPERM) for attributes: stacksize: 1024k, guardsize: 4k, detached.                                                                                                      │
[0.008s][error  ][gc,task  ] Failed to create worker thread                                                                                                                                                                                                       
Stream closed EOF for verrazzano-system/weblogic-operator-webhook-54678fdfb5-2kq22 (weblogic-operator-webhook)
```
<img width="2476" alt="Screenshot 2023-04-08 at 5 05 24 PM" src="https://user-images.githubusercontent.com/7238675/230717804-3435325d-dbfa-41a9-ad9d-e905a8305635.png">

`$ docker --version`
```
Docker version 23.0.3, build 3e7cbfd
```
`$ kind version`
```
kind v0.18.0 go1.20.2 linux/amd64
```
kind cluster create command: 
```
kind create cluster --config - <<EOF
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
  - role: control-plane
    image: kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
    kubeadmConfigPatches:
      - |
        kind: ClusterConfiguration
        apiServer:
          extraArgs:
            "service-account-issuer": "kubernetes.default.svc"
            "service-account-signing-key-file": "/etc/kubernetes/pki/sa.key"
    extraMounts:
      - hostPath: /var/lib/docker/volumes/containerd/_data
        containerPath: /var/lib/containerd #This is the location of the image cache inside the Kind container
EOF
```

**Comment by audacioustux** [comment]
https://github.com/adoptium/containers/issues/215#issuecomment-1142046045 related?

**Comment by ddsharpe** [comment]
@audacioustux we are looking into this and will reply soon.

**Comment by ankedia** [comment]
@audacioustux Can you run `docker version` command to check both client and server docker engine versions? We have not seen this error but it might be related to the older docker engine server version.

$ docker version
Client: Docker Engine - Community
 Version:           19.03.11-ol

Server: Docker Engine - Community
 Engine:
  Version:          19.03.11-ol

**Comment by ankedia** [comment]
I tried installing Verrazzano with nodes image version `1.21.14` (nodes.image) and the  weblogic-operator-webhook pod started successfully. Can you try with a more recent nodes image of kind?

```
verrazzano-system    weblogic-operator-5574fd446-srvr8                                 2/2     Running             0          5m23s
verrazzano-system    weblogic-operator-webhook-54678fdfb5-8f72p                        1/1     Running             0          5m24s
```

**Comment by rjeberhard** [comment]
The Kubernetes 1.21 node image for Kind 0.18.0 is `kindest/node:v1.21.14@sha256:27ef72ea623ee879a25fe6f9982690a3e370c68286f4356bf643467c552a3888` based on the [Kind releases page](https://github.com/kubernetes-sigs/kind/releases/tag/v0.18.0).

**Comment by bryantidd** [comment]
What version of Kubernetes is running?

**Comment by bryantidd** [comment]
Also, What are the resources of the machine being used? Did you review the minimum resources needed?

**Comment by audacioustux** [comment]
sorry for the late feedback... thanks for all the responses

@ankedia 
```
$ docker version
Client: Docker Engine - Community
 Version:           23.0.3
 API version:       1.42
 Go version:        go1.19.7
 Git commit:        3e7cbfd
 Built:             Tue Apr  4 22:05:48 2023
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          23.0.3
  API version:      1.42 (minimum version 1.12)
  Go version:       go1.19.7
  Git commit:       59118bf
  Built:            Tue Apr  4 22:05:48 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.20
  GitCommit:        2806fc1057397dbaeefbea0e4e17bddfbd388f38
 runc:
  Version:          1.1.5
  GitCommit:        v1.1.5-0-gf19387a
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

@rjeberhard 
looks like it's fixed in node:v1.21.14 ... which is the latest k8s version compatible with verrazzano? is that documented somewhere in the docs?

<img width="2424" alt="Screenshot 2023-04-13 at 12 51 03 PM" src="https://user-images.githubusercontent.com/7238675/231677300-1468871b-784f-42c7-834f-b99ff431fc2f.png">
haven't tested anything yet, so i'll close this issue tommorrow

@bryantidd yes
<img width="1358" alt="Screenshot 2023-04-13 at 12 49 42 PM" src="https://user-images.githubusercontent.com/7238675/231677107-930c0665-b698-4ecb-932d-b3131464895b.png">

**Comment by absBitm** [comment]
@audacioustux 
Please check on https://verrazzano.io/latest/docs/setup/prereqs/
